app/src/main/java/com/example/composetemplate/presentation/screens/entry_screens/register/AuthViewModel.kt

```diff
@@ -40,11 +40,18 @@ class AuthViewModel(
     private val _isProgressVisible = mutableStateOf(false)
     val isProgressVisible: Boolean by _isProgressVisible
 
+    //forgot password dialog visibility
+    private val _isForgotDialogVisible = mutableStateOf(false)
```

Comment:
Tom told us there is new cleaner syntax for those cases of mutable and immutable if you want I think it will be nice to learn that